### PR TITLE
To solve failing to compile web projects importing `win_ble`.

### PR DESCRIPTION
**Pull request by chan150**
If a web project has a dependency including `win_ble`, it occur failing to compile.

The solution is that remove the dependencies of `win32` and `ffi` in this library.

### CODE REVIEW

**Comment by rohitsangwan01** [comment]
@chan150 thanks for the pr, why renamed to win_ble_plus ?

**Comment by chan150** [comment]
> @chan150 thanks for the pr, why renamed to win_ble_plus ?

`win_ble_plus` is a temporay package to make web-compilable project.
I set `win_ble_plus` to `discontinued` and `unlisted`.

For this merging, I rename all `win_ble_plus` to `win_ble` in code.

File: lib/win_ble.dart [hunk 1]
```diff
@@ -1,4 +1,4 @@
-library win_ble;
+library win_ble_plus;
```
Comment (rohitsangwan01) [inline lib/win_ble.dart:1]: please change this as well

**Comment by rohitsangwan01** [comment]
@chan150 can you instead push a new pr with just the path provider update and nothing else, that will be easy to review and merge

File: lib/win_ble.dart (hunk 1 above)
Comment (chan150) [reply lib/win_ble.dart]: Sorry, I fix it.

**Comment by chan150** [comment]
> @chan150 can you instead push a new pr with just the path provider update and nothing else, that will be easy to review and merge

I will make new pr for making easy to review and merge.

**Pull request closed without merging**